app/main.py

```diff
@@ -129,6 +129,20 @@ def create_presigned_url(expiration=3600):
 
     return response
 
+# Reverse proxy for objects from S3, a simple get object
+# operation. This is used by scaffoldvuer and its 
+# important to keep the relative <path> for accessing
+# other required files.
+@app.route("/object-resource/<path:path>")
```

Comment:
I would probably have the route called `s3-resource`

Reply:
Ok.